tests/codebuild/run_canarytest.sh

```diff
@@ -52,6 +59,7 @@ echo "Launching canary test for ${COMMIT_SHA}"
 echo "Launching the cluster"
 
 cluster_name="sagemaker-k8s-canary-"$(date '+%Y-%m-%d-%H-%M-%S')""
+crd_namespace="sagemaker-k8s-operator-system"
```

Comment:
We probably want to run the CRDs in a namespace different to that of the controller. `default` is usually best because we are guaranteed to have admin privileges.

Reply:
yep variable is named badly, I have cleaned up the code. It was not being used to install CRD